From 7227801a8b2b963b2debe57e558fd24b7f8787fd Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Mon, 20 Jan 2014 19:30:09 +0800
Subject: [PATCH 0332/1594] ENGR00295814 ARM: dts: imx6qdl: correct gpio key's
 active state

commit 8075412a87bfba49e8f0dcfc56f2457da4161fb5 from
git://git.freescale.com/imx/linux-2.6-imx.git

From schematic, below GPIO keys' active state is low, so we need
to set correct active state in dts.

i.MX6Q/DL-SABRESD board: power, vol+ and vol-.
i.MX6Q/DL-SABREAUTO board: home, back, prog, vol+ and vol-.

Signed-off-by: Anson Huang <b20788@freescale.com>
(cherry picked from commit f1319b3268db3e0e80d85ba9f4ae3b569b916dd4)
Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index 2fd519f..180c332 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -51,35 +51,35 @@
 
 		home {
 			label = "Home";
-			gpios = <&gpio1 11 0>;
+			gpios = <&gpio1 11 1>;
 			gpio-key,wakeup;
 			linux,code = <KEY_HOME>;
 		};
 
 		back {
 			label = "Back";
-			gpios = <&gpio1 12 0>;
+			gpios = <&gpio1 12 1>;
 			gpio-key,wakeup;
 			linux,code = <KEY_BACK>;
 		};
 
 		program {
 			label = "Program";
-			gpios = <&gpio2 12 0>;
+			gpios = <&gpio2 12 1>;
 			gpio-key,wakeup;
 			linux,code = <KEY_PROGRAM>;
 		};
 
 		volume-up {
 			label = "Volume Up";
-			gpios = <&gpio2 15 0>;
+			gpios = <&gpio2 15 1>;
 			gpio-key,wakeup;
 			linux,code = <KEY_VOLUMEUP>;
 		};
 
 		volume-down {
 			label = "Volume Down";
-			gpios = <&gpio5 14 0>;
+			gpios = <&gpio5 14 1>;
 			gpio-key,wakeup;
 			linux,code = <KEY_VOLUMEDOWN>;
 		};
-- 
1.7.5.4

